core/src/main/resources/jenkins/model/Jenkins/help-rawBuildsDir.html

```diff
@@ -1,15 +1,21 @@
 <div>
-    Specify where Jenkins would store records of the past builds.
-    This value can include the following variables.
-
+    Specify where Jenkins should store build records. This includes the console
+    output and other metadata generated by a build.
+    <p>
+    This value can include the following variables:
     <ul>
-        <li><tt>${JENKINS_HOME}</tt> &mdash; Jenkins home directory.
-        <li><tt>${ITEM_ROOTDIR}</tt> &mdash; Root directory of a job for which the default workspace is allocated.
-        <li><tt>${ITEM_FULL_NAME}</tt> &mdash; '/'-separated job name, like "foo/bar".
+        <li><tt>${JENKINS_HOME}</tt> &mdash; The Jenkins home directory
+        <li><tt>${ITEM_ROOTDIR}</tt> &mdash; The directory where Jenkins stores config data for a given job
```

Comment:
this is still the root dreictory where jenkins stores the config.

e.g. a job foo/bar/whizz will not be stored in this directory but a subdirectory of it.